schema.sql

```diff
@@ -65,3 +64,25 @@ CREATE TABLE visits (
     CONSTRAINT unique_visit_animal_vet UNIQUE (animal_id, vet_id, visit_date)
 );
 
+ALTER TABLE visits ADD COLUMN date_of_visit timestamp;
+
+SELECT COUNT(*) FROM visits WHERE animal_id = 4;
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4;
+
+
+SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4;
+
+
+SELECT * FROM visits where vet_id = 2;
+SELECT * FROM owners where email = 'owner_18327@mail.com';
+
+CREATE INDEX idx_animal_id ON visits(animal_id);
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4;
```

Comment:
- [ ] looking at the screenshot for the `EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4;` before adding an index, the `Planning and Execution time` is below `1000ms`. According to the requirement, you need to keep populating the database with the data provided until the `Execussion time is above `1000ms`, then you work on it to reduce it. I'll suggest you take this into consideration/. :+1: :100: